### ISSUE TITLE
Pufftagg

### ISSUE THREAD

**Issue by ErikXXon** [victorjonsson/Arlima#96]
What is an "pufftagg"?

**Comment by victorjonsson** [comment]
Where is the name used?

**Comment by ErikXXon** [comment]
In the settings for Arlima

![pufftagg](https://cloud.githubusercontent.com/assets/6995808/7174062/e0fc64f6-e401-11e4-9d1c-8ec2cfbc9546.PNG)

**Comment by victorjonsson** [comment]
ping @aaslun Can you provide us with some info on this :) I know what it is but I'm unsure how to display the "pufftagg" in the template.

**Comment by aaslun** [comment]
I guess you can say that "pufftagg" (or newsbill tag, as it was translated to) is local newspaper jargon. It is a pre-lead in word or sentence that is often used to frame the "essence" of the article. I don't think that it is used in any of the default Arlima templates so if you want to use it you can extract it in the `arlima_template_object`-filter under the object's `['options']['newsbillTag']` index.

![pufftagg](https://cloud.githubusercontent.com/assets/555273/7727899/2b3cb406-ff0c-11e4-9ddf-db78a3fbbd2c.jpg)

**Comment by victorjonsson** [comment]
Okey! In that case you can access it directly in the mustache templates using `{{{options.newsbillTag}}}`

```html
{{#options.newsbillTag}}
   <a href="{{{url}}}" class="newsbill-tag">
      {{{options.newsbillTag}}}
   </a>
{{/options.newsbillTag}}
```

**Comment by ErikXXon** [comment]
Thanks!

**Comment by ErikXXon** [comment]
If i want to use newsbill tag, but dont want to replace the fontsize control - Is there an option for that?

**Comment by victorjonsson** [comment]
Yes you can use the entry-word button in the wysiwyg-editor. It's the first button with a symbol like ►Ⅰ. It will become a link with the class `teaser-entryword` on front end.
